packages/fxa-auth-server/lib/routes/oauth/destroy.js

```diff
@@ -60,7 +59,7 @@ module.exports = ({ log, oauthDB }) => {
       // Log a warning if legacy client_secret is provided, so we can
       // measure whether it's safe to remove this behaviour.
       log.warn('destroy.unexpectedClientSecret', {
-        client_id: hex(token.clientId),
+        client_id: token.clientId.toString('hex'),
```

Comment:
Not that we should fix this in this PR, but is their a reason the database can't just make sure the clientId is a hexstring? I can't think of areason we wouldn't want retain the buffer, but maybe there is a case where it matters.